server/utils/applicationUtils.ts

```diff
@@ -21,13 +21,22 @@ const dashboardTableRows = (applications: Array<Application>): Array<TableRow> =
       textValue(application.person.crn),
       htmlValue(tierBadge(application.risks.tier.value?.level || '')),
       textValue(arrivalDate ? DateFormats.isoDateToUIDate(arrivalDate, { format: 'short' }) : 'N/A'),
-      textValue(
-        application.submittedAt ? DateFormats.isoDateToUIDate(application.submittedAt, { format: 'short' }) : 'N/A',
-      ),
+      htmlValue(getStatus(application)),
     ]
   })
 }
 
+const getStatus = (application: Application): string => {
+  switch (application.status) {
```

Comment:
The statuses might not line up nicely but we have a `getStatus` function for assessments that we could potentially add these statuses too as well. Depends how DRY you want to go!

Reply:
I think it's better to separate them out, especially as you'd have to do a bunch of typechecking too